transformations/ocr_perturbation/README.md

```diff
@@ -0,0 +1,83 @@
+# OCR Perturbation 🦎+ ⌨️ → 🐍
+This transformation directly induces Optical Character Recognition (OCR) errors into the input text. It renders the input sentence as an image and recognizes the rendered text using an OCR engine:
+
+<p align="center"> 
+  <b>Neil Alden Armstrong was an American astronaut.</b>
+  <br>
+    <font size="+3">&#8595;</font>
+  <br>
+  <img src="example.png" style="zoom:200%;" />
+  <br>
+    <font size="+3">&#8595;</font>
+  <br>
+    <font size="+3">&#128441;&#128269;</font><font size="+4" color="blue">Tesseract 4 OCR</font>
+  <br>
+    <font size="+3">&#8595;</font>
+  <br>
+    <b>Neil: Alden Armstrong was an 'Ametican astronaut:</b>
+</p>
+
+
+Author: [Marcin Namysl](https://github.com/mnamysl/)
+
+## What type of transformation is this?
+This transformation acts like a sentence-level perturbation. Multiple variations can be created via changing parameters, e.g., various image rendering options can be specified.
+
+This transformation employs the [trdg](https://pypi.org/project/trdg/) package for image rendering and the [tesserocr](https://pypi.org/project/tesserocr/) package for OCR. Currently, it can be applied to text in English, French, Spanish, or German.
+
+#### Prerequisites 
+
+1) Download [spaCy](https://pypi.org/project/spacy/) models for the supported languages:
+
+```sh
+python -m spacy download en_core_web_sm
+python -m spacy download fr_core_web_sm
+python -m spacy download es_core_web_sm
+python -m spacy download de_core_web_sm
+```
+2. Install [Tesseract](https://github.com/tesseract-ocr/tesseract) (including dependencies and models). See the [installation instructions](https://tesseract-ocr.github.io/tessdoc/#compiling-and-installation) for your operating system. For example, On Debian or Ubuntu, you can simply use its developer tools:
+
+```bash
+apt-get install tesseract-ocr libtesseract-dev libleptonica-dev pkg-config \
+  tesseract-ocr-eng tesseract-ocr-osd tesseract-ocr-deu tesseract-ocr-fra tesseract-ocr-spa
+```
```

Comment:
Silly question: Is this step enough for the transformation to work or should a user also pip install requirements.txt?

Reply:
Python dependencies should also be installed, thank you for spotting this inconsistency!

I updated the instructions in the README.md file accordingly.

Reply:
Great!